src/qibo/transpiler/decompositions.py

```diff
@@ -402,6 +402,38 @@ def _u3_to_gpi2(t, p, l):
 )
 
 
+def _decomposition_generalized_RBS(ins, outs, theta, phi, controls):
```

Comment:
Adding here some docstrings could improve the readability for who is not familiar with the paper.

Reply:
The docstring would be basically redundant w.r.t. the one of `gates.GeneralizedRBS`, which is why I chose to not repeat it.